Bryn Mawr Classical Review.js

```diff
@@ -59,6 +76,8 @@ function doWeb(doc, url) {
 	Zotero.Utilities.processDocuments(arts, function(doc) {
 		var item = new Zotero.Item("journalArticle");
 		var title = doc.evaluate('//h3/i', doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
```

Comment:
I would suggest to just use the functions `ZU.xpath` and `ZU.xpathText` for that.